src/components/Form/Form.js

```diff
@@ -0,0 +1,25 @@
+import useStore from '../../hooks/useStore';
+
+import StyledForm from './StyledForm';
+
+export default function Form() {
+	const addTask = useStore(state => state.addTask);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+		const name = event.target.elements.task.value.trim();
+		const duration = Number.parseInt(event.target.elements.duration.value, 10);
+		addTask(name, duration);
+		event.target.reset();
```

Comment:
Nice! You can also write it like this:

```suggestion
        const form = event.target
		const name = form.elements.task.value.trim();
		const duration = Number.parseInt(form.elements.duration.value, 10);
		addTask(name, duration);
		form.reset();
```

Maybe in this way it is a bit easier to read 🙂